test cases/failing/46 pkgconfig variables zero length value/meson.build

```diff
@@ -0,0 +1,16 @@
+project('variables-zero-length-test', 'c', version : '1.0')
```

Comment:
Oh sorry I realize they are *failing* tests so nevermind you can't really have multiple in one file.